src/components/Editor/Menu/Bubble/index.jsx

```diff
@@ -78,9 +78,7 @@ const Bubble = ({
             tooltips,
           }}
           options={textOptions}
-        >
-          {children}
```

Comment:
The children is not passed to any of the menu component variant. Looks useless and removed for all components.